### PR TITLE
[blog] Adding a new community update blogpost.

### PR DESCRIPTION
**Pull request by morsapaes**
Community update for May'20 covering the most recent milestones and developments in the Flink community.

### CODE REVIEW

File: _posts/2020-05-06-community-update.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+	
+# The Bigger Picture
+
+## A new self-paced Apache Flink training
+
+<div style="line-height:60%;">
+    <br>
+</div>
+
+This week, the Flink website received the contribution of a self-paced training course curated by David ([@alpinegizmo](https://twitter.com/alpinegizmo)). The new materials guide you through the very basics of Flink and the DataStream API, and round off every concepts section with hands-on exercises to help you better assimilate what you learned.
```
Comment (MarkSfik) [inline _posts/2020-05-06-community-update.md]:
```suggestion
This week, the Flink website received the contribution of a self-paced training course curated by David ([@alpinegizmo](https://twitter.com/alpinegizmo)). The new materials guide you through the very basics of Flink and the DataStream API, and round off every concept section with hands-on exercises to help you better assimilate what you learned.
```

**Review by MarkSfik — COMMENTED** [review]
That's a great post! 
Thanks for writing this @morsapaes

File: _posts/2020-05-06-community-update.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+Last month also marked the debut of Flink Forward Virtual Conference 2020: what did you think? If you missed it altogether or just want to recap some of the sessions, the [videos](https://www.youtube.com/playlist?list=PLDX4T_cnKjD0ngnBSU-bYGfgVv17MiwA7) and [slides](https://www.slideshare.net/FlinkForward) are now available!
+
+{% toc %}
+
+# The Past Month in Flink
+
+## Flink Stateful Functions 2.0 is out!
+
+In the beginning of April, the Flink community announced the [release of Stateful Functions 2.0](https://flink.apache.org/news/2020/04/07/release-statefun-2.0.0.html) — the first as part of the Apache Flink project. From this release, you can use Flink as the base of a (stateful) serverless platform with out-of-the-box consistent and scalable state, and efficient messaging between functions. You can even run your stateful functions on platforms like AWS Lambda, as Gordon ([@tzulitai](https://twitter.com/tzulitai)) demonstrated in [his Flink Forward talk](https://www.youtube.com/watch?v=tuSylBadNSo&list=PLDX4T_cnKjD0ngnBSU-bYGfgVv17MiwA7&index=27&t=8s).
```
Comment (rmetzger) [inline _posts/2020-05-06-community-update.md:24]:
```suggestion
In the beginning of April, the Flink community announced the [release of Stateful Functions 2.0](https://flink.apache.org/news/2020/04/07/release-statefun-2.0.0.html) — the first as part of the Apache Flink project. From this release, you can use Flink as a (stateful) serverless platform with out-of-the-box consistent and scalable state, and efficient messaging between functions. You can even run your stateful functions on platforms like AWS Lambda, as Gordon ([@tzulitai](https://twitter.com/tzulitai)) demonstrated in [his Flink Forward talk](https://www.youtube.com/watch?v=tuSylBadNSo&list=PLDX4T_cnKjD0ngnBSU-bYGfgVv17MiwA7&index=27&t=8s).
```

File: _posts/2020-05-06-community-update.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+
+<center>
+<img src="{{ site.baseurl }}/img/blog/2020-05-06-community-update/2020-05-06-community-update_2.png" width="550px" alt="Stateful Functions"/>
+</center>
+
+<div style="line-height:60%;">
+    <br>
+</div>
+
+It’s been encouraging to see so many questions about Stateful Functions popping up on the Mailing List and Stack Overflow! If you’d like to get involved, we’re always [looking for new contributors](https://github.com/apache/flink-statefun#contributing) — especially around SDKs for other languages like Go, Javascript and Rust.
```
Comment (rmetzger) [inline _posts/2020-05-06-community-update.md]:
```suggestion
It’s been encouraging to see so many questions about Stateful Functions popping up on [the Mailing List](https://lists.apache.org/list.html?user@flink.apache.org:lte=3M:statefun) and Stack Overflow! If you’d like to get involved, we’re always [looking for new contributors](https://github.com/apache/flink-statefun#contributing) — especially around SDKs for other languages like Go, Javascript and Rust.
```

File: _posts/2020-05-06-community-update.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+<hr>
+
+## Warming up for Flink 1.11
+
+The final preparations for the release of Flink 1.11 are well underway and there’s a lot of new features and improvements to look out for:
```
Comment (rmetzger) [inline _posts/2020-05-06-community-update.md]:
```suggestion
The final preparations for the release of Flink 1.11 are well underway, with the feature freeze scheduled for May 15th and there’s a lot of new features and improvements to look out for:
```

File: _posts/2020-05-06-community-update.md (hunk 1 above)
Comment (morsapaes) [reply _posts/2020-05-06-community-update.md]: I used an attributive noun on purpose here and personally prefer that.

File: _posts/2020-05-06-community-update.md [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+
+* On the **usability** side, you can expect a big focus on smoothing data ingestion with contributions like support for changelog streams in the Table API/SQL ([FLIP-105](https://cwiki.apache.org/confluence/display/FLINK/FLIP-105%3A+Support+to+Interpret+and+Emit+Changelog+in+Flink+SQL)), easy streaming data ingestion into Apache Hive ([FLIP-115](https://cwiki.apache.org/confluence/display/FLINK/FLIP-115%3A+Filesystem+connector+in+Table)) or support for Pandas DataFrames in PyFlink ([FLIP-120](https://cwiki.apache.org/confluence/display/FLINK/FLIP-120%3A+Support+conversion+between+PyFlink+Table+and+Pandas+DataFrame)). 
+
+* On the **operational** side, the much anticipated new Source API ([FLIP-27](https://cwiki.apache.org/confluence/display/FLINK/FLIP-27%3A+Refactor+Source+Interface)) will unify batch and streaming sources, and improve out-of-the-box event-time behavior; while unaligned checkpoints ([FLIP-76](https://cwiki.apache.org/confluence/display/FLINK/FLIP-76%3A+Unaligned+Checkpoints)) and some changes to network memory management will allow to speed up checkpointing under backpressure.
```
Comment (rmetzger) [inline _posts/2020-05-06-community-update.md]:
```suggestion
* On the **operational** side, the much anticipated new Source API ([FLIP-27](https://cwiki.apache.org/confluence/display/FLINK/FLIP-27%3A+Refactor+Source+Interface)) will unify batch and streaming sources, and improve out-of-the-box event-time behavior; while unaligned checkpoints ([FLIP-76](https://cwiki.apache.org/confluence/display/FLINK/FLIP-76%3A+Unaligned+Checkpoints)) and changes to network memory management will allow to speed up checkpointing under backpressure.
```

File: _posts/2020-05-06-community-update.md [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+
+It’s been encouraging to see so many questions about Stateful Functions popping up on the Mailing List and Stack Overflow! If you’d like to get involved, we’re always [looking for new contributors](https://github.com/apache/flink-statefun#contributing) — especially around SDKs for other languages like Go, Javascript and Rust.
+
+<hr>
+
+## Warming up for Flink 1.11
+
+The final preparations for the release of Flink 1.11 are well underway and there’s a lot of new features and improvements to look out for:
+
+* On the **usability** side, you can expect a big focus on smoothing data ingestion with contributions like support for changelog streams in the Table API/SQL ([FLIP-105](https://cwiki.apache.org/confluence/display/FLINK/FLIP-105%3A+Support+to+Interpret+and+Emit+Changelog+in+Flink+SQL)), easy streaming data ingestion into Apache Hive ([FLIP-115](https://cwiki.apache.org/confluence/display/FLINK/FLIP-115%3A+Filesystem+connector+in+Table)) or support for Pandas DataFrames in PyFlink ([FLIP-120](https://cwiki.apache.org/confluence/display/FLINK/FLIP-120%3A+Support+conversion+between+PyFlink+Table+and+Pandas+DataFrame)).
```
Comment (rmetzger) [inline _posts/2020-05-06-community-update.md]:
```suggestion
* On the **usability** side, you can expect a big focus on smoothing data ingestion with contributions like support for changelog (CDC) streams in the Table API/SQL ([FLIP-105](https://cwiki.apache.org/confluence/display/FLINK/FLIP-105%3A+Support+to+Interpret+and+Emit+Changelog+in+Flink+SQL)), easy streaming data ingestion into Apache Hive ([FLIP-115](https://cwiki.apache.org/confluence/display/FLINK/FLIP-115%3A+Filesystem+connector+in+Table)) or support for Pandas DataFrames in PyFlink ([FLIP-120](https://cwiki.apache.org/confluence/display/FLINK/FLIP-120%3A+Support+conversion+between+PyFlink+Table+and+Pandas+DataFrame)). 
```

**Comment by rmetzger** [comment]
Nice blog post, thank you! I left some comments, once they are addressed, I think it generally good to go.

The section about the release and training could be a bit more exciting:
- for the release, we could stress that the Python API has really matured now (metrics, better udf support); unaligned checkpoints is a really big effort that needed a lot of thought and is a non-trivial change, helping many of our power users.
- For the training, it is the entire training material of Ververica, now licensed under apache license, as part of apache. We should maybe somehow stress that this is a big, valuable contribution (and that there are efforts underway of translating it to Chinese (if I understand the PRs correctly))

**Comment by morsapaes** [comment]
Thanks a lot for the review, @rmetzger and @MarkSfik!

I addressed most comments in the latest commit. Thanks for the remarks on 1.11, it really deserves a bit more enthusiasm! 👍 For the training, I wouldn't compromise on mentioning that there are active efforts for the chinese translation, since it seems like so far only a ticket opened by David exists (as far I understand from the contribution PR and the mailing list thread).

**Comment by sjwiesman** [comment]
Looks good! Merging . . .

**Pull request merged**